api/service/api.proto

```diff
@@ -3,22 +3,20 @@ syntax = "proto3";
 package api;
 option go_package = "service";
 
-import "github.com/mesg-foundation/core/service/service.proto";
-
 service Service {
   rpc EmitEvent (EmitEventRequest) returns (EmitEventReply) {}
   rpc ListenTask (ListenTaskRequest) returns (stream TaskData) {}
   rpc SubmitResult (SubmitResultRequest) returns (SubmitResultReply) {}
 }
 
 message EmitEventRequest {
-  service.Service service = 1;
+  string serviceHash = 1;
```

Comment:
Why naming it `ServiceHash`? We already have `ServiceID` in the core api.

Reply:
the idea is that this hash can be something else than the id of the service, can be the public address or stuff like that, I was trying to avoid `serviceID` to not be confusing because how the developer can have the id when the id is generated when you deploy the service.

For now we use the service ID as identifier but we might change that with a random string created when you start the service or a uuid or anything like that but yes maybe it should be more generic like "SECRET" or "AUTHORIZATION" or "TOKEN" stuff like that what do you think @NicolasMahe